pkg/config/engine/containerd/config_v1.go

```diff
@@ -22,13 +22,25 @@ import (
 	"github.com/pelletier/go-toml"
 
 	"github.com/NVIDIA/nvidia-container-toolkit/pkg/config/engine"
+	cfgtoml "github.com/NVIDIA/nvidia-container-toolkit/pkg/config/toml"
 )
 
 // ConfigV1 represents a version 1 containerd config
 type ConfigV1 Config
 
 var _ engine.Interface = (*ConfigV1)(nil)
 
+type ctrdCfgV1Runtime struct {
```

Comment:
Can we add the below to ensure this new type adheres to the engine.RuntimeConfig interface?
```
var _ engine.RuntimeConfig = (*ctrdCfgV1Runtime)(nil)
```